src/modules/NotFoundPage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import {Stack, Typography} from '@mui/material';
+import ReportOutlinedIcon from '@mui/icons-material/ReportOutlined';
+
+
+const NotFoundPage = (): JSX.Element => {
+
+  return (
+    <Stack spacing={2} sx={{height: '50vw', alignItems: 'center', justifyContent: 'center'}}>
+      <ReportOutlinedIcon sx={{color: '#E5EDF2',width: 200,height: 200}}/>
+      <Typography
+        variant={'h1'}
```

Comment:
Unnecessary curly brackets. Change everywhere.

```suggestion
        variant="h1"
```